### ISSUE TITLE
undefined local variable or method `its' in 0.2.2

### ISSUE THREAD

**Issue by andrew** [SchemaPlus/schema_plus_indexes#3]
Looks like removing the its-it dependency in https://github.com/ronen/modware/pull/2 has broken `schema_plus_pg_indexes` for me on https://github.com/andrew/contribulator, see this travis build: https://github.com/andrew/contribulator

Adding `its-it` back to the Gemfile file gets the tests passing again: https://github.com/andrew/contribulator/pull/295

```
== 20160421143043 AddIndexesToProjects: migrating =============================
-- change_table(:projects)
   -> 0.0090s
== 20160421143043 AddIndexesToProjects: migrated (0.0091s) ====================
rake aborted!
NameError: undefined local variable or method `its' for #<Modware::Stack::Middleware:0x00000005cce7f0>
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/schema_plus_indexes-0.2.2/lib/schema_plus/indexes/middleware/dumper.rb:14:in `block in after'
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/schema_plus_indexes-0.2.2/lib/schema_plus/indexes/middleware/dumper.rb:11:in `each'
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/schema_plus_indexes-0.2.2/lib/schema_plus/indexes/middleware/dumper.rb:11:in `after'
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/modware-0.1.3/lib/modware/stack.rb:46:in `block in execute'
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/modware-0.1.3/lib/modware/stack.rb:45:in `each'
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/modware-0.1.3/lib/modware/stack.rb:45:in `execute'
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/modware-0.1.3/lib/modware/stack.rb:19:in `start'
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/schema_monkey-2.1.5/lib/schema_monkey/stack.rb:34:in `start'
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/schema_plus_core-1.0.1/lib/schema_plus/core/active_record/schema_dumper.rb:48:in `table'
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/activerecord-4.2.6/lib/active_record/schema_dumper.rb:97:in `block in tables'
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/activerecord-4.2.6/lib/active_record/schema_dumper.rb:96:in `each'
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/activerecord-4.2.6/lib/active_record/schema_dumper.rb:96:in `tables'
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/schema_plus_core-1.0.1/lib/schema_plus/core/active_record/schema_dumper.rb:43:in `block in tables'
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/modware-0.1.3/lib/modware/stack.rb:54:in `call_implementation'
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/modware-0.1.3/lib/modware/stack.rb:84:in `_modware_continue'
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/modware-0.1.3/lib/modware/stack.rb:76:in `_modware_call'
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/modware-0.1.3/lib/modware/stack.rb:43:in `execute'
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/modware-0.1.3/lib/modware/stack.rb:19:in `start'
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/schema_monkey-2.1.5/lib/schema_monkey/stack.rb:34:in `start'
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/schema_plus_core-1.0.1/lib/schema_plus/core/active_record/schema_dumper.rb:42:in `tables'
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/activerecord-4.2.6/lib/active_record/schema_dumper.rb:38:in `dump'
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/schema_plus_core-1.0.1/lib/schema_plus/core/active_record/schema_dumper.rb:17:in `dump'
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/activerecord-4.2.6/lib/active_record/schema_dumper.rb:22:in `dump'
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/activerecord-4.2.6/lib/active_record/railties/databases.rake:240:in `block (4 levels) in <top (required)>'
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/activerecord-4.2.6/lib/active_record/railties/databases.rake:239:in `open'
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/activerecord-4.2.6/lib/active_record/railties/databases.rake:239:in `block (3 levels) in <top (required)>'
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/activerecord-4.2.6/lib/active_record/railties/databases.rake:52:in `block (2 levels) in <top (required)>'
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/activerecord-4.2.6/lib/active_record/railties/databases.rake:45:in `block (2 levels) in <top (required)>'
/home/travis/build/andrew/contribulator/vendor/bundle/ruby/2.3.0/gems/rake-11.2.2/exe/rake:27:in `<top (required)>'
/home/travis/.rvm/gems/ruby-2.3.1@global/gems/bundler-1.12.5/lib/bundler/cli/exec.rb:63:in `load'
/home/travis/.rvm/gems/ruby-2.3.1@global/gems/bundler-1.12.5/lib/bundler/cli/exec.rb:63:in `kernel_load'
/home/travis/.rvm/gems/ruby-2.3.1@global/gems/bundler-1.12.5/lib/bundler/cli/exec.rb:24:in `run'
/home/travis/.rvm/gems/ruby-2.3.1@global/gems/bundler-1.12.5/lib/bundler/cli.rb:304:in `exec'
/home/travis/.rvm/gems/ruby-2.3.1@global/gems/bundler-1.12.5/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/home/travis/.rvm/gems/ruby-2.3.1@global/gems/bundler-1.12.5/lib/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
/home/travis/.rvm/gems/ruby-2.3.1@global/gems/bundler-1.12.5/lib/bundler/vendor/thor/lib/thor.rb:359:in `dispatch'
/home/travis/.rvm/gems/ruby-2.3.1@global/gems/bundler-1.12.5/lib/bundler/vendor/thor/lib/thor/base.rb:440:in `start'
/home/travis/.rvm/gems/ruby-2.3.1@global/gems/bundler-1.12.5/lib/bundler/cli.rb:11:in `start'
/home/travis/.rvm/gems/ruby-2.3.1@global/gems/bundler-1.12.5/exe/bundle:27:in `block in <top (required)>'
/home/travis/.rvm/gems/ruby-2.3.1@global/gems/bundler-1.12.5/lib/bundler/friendly_errors.rb:98:in `with_friendly_errors'
/home/travis/.rvm/gems/ruby-2.3.1@global/gems/bundler-1.12.5/exe/bundle:19:in `<top (required)>'
/home/travis/.rvm/gems/ruby-2.3.1/bin/bundle:23:in `load'
/home/travis/.rvm/gems/ruby-2.3.1/bin/bundle:23:in `<main>'
/home/travis/.rvm/gems/ruby-2.3.1/bin/ruby_executable_hooks:15:in `eval'
/home/travis/.rvm/gems/ruby-2.3.1/bin/ruby_executable_hooks:15:in `<main>'
Tasks: TOP => db:schema:dump
(See full trace by running task with --trace)
```

**Comment by ronen** [comment]
d'oh!  missing require.  released the fix in v0.2.3

**Comment by andrew** [comment]
Great, thanks!
